pkg/build/controller/strategy/custom.go

```diff
@@ -125,5 +125,6 @@ func (bs *CustomBuildStrategy) CreateBuildPod(build *buildapi.Build) (*v1.Pod, e
 	setupSourceSecrets(pod, &pod.Spec.Containers[0], build.Spec.Source.SourceSecret)
 	setupInputSecrets(pod, &pod.Spec.Containers[0], build.Spec.Source.Secrets)
 	setupAdditionalSecrets(pod, &pod.Spec.Containers[0], build.Spec.Strategy.CustomStrategy.Secrets)
+	setupAdditionalConfigMaps(pod, &pod.Spec.Containers[0], build.Spec.Strategy.CustomStrategy.ConfigMaps)
```

Comment:
so doesn't this need to be doing a "setupInputConfigMaps" also?

Reply:
yes (had it in my prior branch, missed carrying it over)